addons/markers/stringtable.xml

```diff
@@ -7,6 +7,7 @@
                 <German>Timi007, PhILoX, Bix</German>
                 <Chinese>Timi007，PhILoX，Bix</Chinese>
                 <Chinesesimp>Timi007，PhILoX，Bix</Chinesesimp>
+                <Korean>MINPD</Korean>
```

Comment:
```suggestion
                <Korean>Timi007，PhILoX，Bix</Korean>
```
I will insert a new Contribution list into the Readme for you and the Chinese translator

Reply:
Check #30